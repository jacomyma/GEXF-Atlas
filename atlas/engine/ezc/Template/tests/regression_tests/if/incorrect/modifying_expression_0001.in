{* file: if/incorrect/modifying_expression_0001.in *}
{var $foo = 42, $bar = 101}
0001:{if $foo = 5}this should never appear{/if}
