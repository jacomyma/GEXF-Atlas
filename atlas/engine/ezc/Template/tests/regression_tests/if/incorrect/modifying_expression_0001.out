mock:3:18: An expression that modifies a variable is not allowed.

0001:{if $foo = 5}this should never appear{/if}
                 ^
